cost/flexera/cco/cheaper_regions/cheaper_regions.pt

```diff
@@ -244,44 +256,79 @@ script "js_new_costs_request", type: "javascript" do
   EOS
 end
 
-script "js_merge_bc_name", type: "javascript" do
-  parameters "new_bc_costs", "billing_centers", "ds_cloud_vendor_accounts"
+datasource "ds_new_bc_cost_name" do
+  run_script $js_new_bc_cost_name, $ds_new_bc_costs, $ds_billing_centers, $ds_cloud_vendor_accounts, $ds_aws_regions_reference, $ds_azure_regions_reference, $ds_google_regions_reference, $ds_applied_policy
+end
+
+script "js_new_bc_cost_name", type: "javascript" do
+  parameters "new_bc_costs", "billing_centers", "ds_cloud_vendor_accounts", "ds_aws_regions_reference", "ds_azure_regions_reference", "ds_google_regions_reference", "ds_applied_policy"
   result "results"
   code <<-EOS
     var results = []
-    var azure_cheaper_regions = {
-      "regions": {
-        "US West":    "US West 2",
-        "US East":    "US East 2",
-        "KR Central": "KR South",
-        "US Central": "US West 2",
-        "CA Central": "CA East",
-        "EU West":    "UK South",
-        "EU North":   "UK West",
-        "JA East":    "KR South",
-        "JA West":    "KR South"
+    var azure_supported_regions = [
+      { region: "westus", cheaper: "westus2" },
```

Comment:
This kind of data should be stored in the regions.json files for each region. For example, in data/azure/regions.json we can do something like this:

{
    "name": "US West",
    "region": "westus",
    "cheaper": "westus2"
}
  
The general idea is that we should not be hardcoding data like this into policies. Instead, policies should reference one source of truth for this data so that, if it needs to be updated, we only need to update it in one place.